server/templates/staff/course/section/section.html

```diff
@@ -0,0 +1,56 @@
+{% extends "staff/base.html" %}
+{% import 'staff/_formhelpers.html' as forms with context %}
+
+
+{% block title %} Section Console - {{ current_course.display_name_with_semester }}{% endblock %}
+
+{% block main %}
+<section class="content-header">
+    <h1>
+        {{ current_course.display_name_with_semester }} Section Console
+        <small>{{ current_course.offering }}</small>
+    </h1>
+    <ol class="breadcrumb">
+        <li><a href="{{ url_for(".course", cid=current_course.id) }}">
+            <i class="fa fa-university"></i> {{ current_course.offering }}
+        </a></li>
+        <li class="active"><a href="{{ url_for('.enrollment', cid=current_course.id) }}">
+            <i class="fa fa-list"></i>Section Console</a>
+        </li>
+    </ol>
+</section>
+<section class="content">
+    {% include 'alerts.html' %}
+    <div class="row">
+            {% for roster, title in ((enrollments, "Your Students"),) %}
+                {% if roster %}
+                    {% include 'staff/course/section/section.list.html' %}
+                {% endif %}
+            {% endfor %}
+            {% for roster, title in ((staff, "Staff Section Assignments"),) %}
+                {% if roster %}
+                    {% include 'staff/course/section/section.assignments.list.html' %}
+                {% endif %}
+            {% endfor %}
+            <div class="box collapsed-box">
+            <div class="box-header with-border">
+                <h3 class="box-title">Edit Section Assignment</h3>
```

Comment:
Why is `Name` a field in the `Edit Section` form? The name can be pulled from the user model we should just need the email to change a `User(s)` section. 

Also this form might be better accessed from an individual student's profile page `/admin/course/1/{email}`

<img width="977" alt="screen shot 2018-04-09 at 17 30 04" src="https://user-images.githubusercontent.com/2927722/38529813-0c7671a6-3c1c-11e8-8d60-0277ec4132df.png">